website/orders/templates/orders/account_history.html

```diff
@@ -0,0 +1,69 @@
+{% extends 'tosti/base.html' %}
+{% load static %}
+
+{% block styles %}
+    <link rel="stylesheet" href="{% static 'users/css/centered-container.css' %}"/>
+{% endblock %}
+
+{% block page %}
+    <div class="container-sm">
+        <h1>Account</h1>
+        <p>The details of your account are displayed below</p>
+        <ul class="nav nav-tabs" role="tablist">
+            <li class="nav-item" style="padding-bottom: 0;">
+                <a class="nav-link" id="account-tab" href="{% url 'users:account' %}"
+                   >Personal details</a>
+            </li>
+            <li class="nav-item" style="padding-bottom: 0;">
+                <a class="nav-link active" id="order-history-tab" href="{% url 'orders:history' %}"
+                   >Order history</a>
+            </li>
+            <li class="nav-item" style="padding-bottom: 0;">
+                <a class="nav-link" id="request-history-tab" href="{% url 'thaliedje:history' %}"
```

Comment:
This violates the app independence

Reply:
Yes but how to solve this? This will always happen as we need to create links between the apps.

Reply:
We can solve this with template tags but then still we need to import them in different templates.